packages/services/server/src/supertokens.ts

```diff
@@ -201,14 +203,14 @@ export const backendConfig = (requirements: {
                 version: '1',
                 superTokensUserId: input.userId,
                 externalUserId,
-                email: user.email,
+                email: user.emails[0],
```

Comment:
Since we don't use the enterprise version each user will always only have a single email.